### ISSUE TITLE
Duplicate of HVAC entities?

### ISSUE THREAD

**Issue by erikbosch** [GENIVI/vehicle_signal_specification#229]
When looking in the specs I found that we have:
```

- IsRecirculationActive:
  datatype: boolean
  type: actuator
  description: Is recirculation active.
```
... in HVAC.vspec, but then also ...

```
#include HVAC.vspec HVAC

- HVAC.IsRecirculationActive:
  datatype: boolean
  type: actuator
  description: Is recirculation active.
```

... in Cabin.vspec. This seems to me to be a duplicate? Should IsRecirculationActive (and similar) be removed from Cabin.vspec

(But as far as I understand our metamodel this is semantically correct, we just redefine/overwrite IsRecirculationActive which is allowed)

https://github.com/GENIVI/vehicle_signal_specification/blob/9c1b89b762462f6e426cab02647b322f92fd6131/spec/Cabin/Cabin.vspec#L31

**Comment by erikbosch** [comment]
Fixed in #230

**Comment by erikbosch** [comment]
PR merged, closing it
